### PR TITLE
Feature/single sighting page

### PR DESCRIPTION
**Pull request by krdurrett**
____ Wrote Tests __x__ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [x] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: Added functionality to single sighting page.
    
## Requested feedback: 
    
    I'd like feedback on... n/a

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const App = () => {
             <Route path="/" element={<Home />} />
             <Route path="/singlecryptid/:name" element={<SingleCryptid />} />
             <Route path="/allsightings" element={<AllSightings />} />
-            <Route path="/singlesighting/:name" element={<SingleSighting />} />
+            <Route path="/singlesighting/:name/:id" element={<SingleSighting />} />
```
Comment (GraceGardner) [inline src/App.js:26]: Such a great idea!!!

**Review by jasmyn2244 — APPROVED** [review]
Clever problem solving! Looks great!

File: src/Components/SingleSighting.js [hunk 2]
```diff
@@ -1,76 +1,65 @@
...
+        <img src={data.sightingById.image} alt='bigfoot' className='sighting-photo' />
         <div className='sighting-name-box'>
-          <p className='sighting-name-text'>Bigfoot</p>
+          <p className='sighting-name-text'>{name}</p>
         </div>
       </div>
       <div className='sighting-info-container'>
       <div className='sighting-details'>
-        <p className='sighting-location'>Location: placeholder</p>
+        <p className='sighting-title'>{data.sightingById.title}</p>
```
Comment (GraceGardner) [inline src/Components/SingleSighting.js:56]: We have info!!!

**Review by GraceGardner — APPROVED** [review]
We have info on all of the pages now! Nice work!

**Pull request merged**